sufia-models/app/models/sufia/works/curation_concern/with_editors.rb

```diff
@@ -0,0 +1,45 @@
+module Sufia::Works
```

Comment:
I used it in the spec/jobs/copy_permissions_job_spec.rb, but could do that a different way.  It is the only place that I used it.